.github/ISSUE_TEMPLATE/operational-issues.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Operational issues
+about: Website and hosting-related issues
+title: ''
+labels: site
+assignees: ''
+
+---
+
+<!-- This issue type is mostly for contributors and developers. If you are seeing an issue with the website and you’re not involved in this project, you probably want a different issue type.
```

Comment:
Meh. Remove this.

Reply:
Remove the entire “operational” issue type, or the HTML comment? Your rationale?